source/developing.html.erb.md

```diff
@@ -175,8 +175,7 @@ Use structural elements to mark-up headings, lists, tables, etc. HTML5 provides
 {:.attach_permalink}
 ## Help users avoid and correct mistakes
 
-
-Be as forgiving of format as possible when processing user input. For example, accept phone numbers that include spaces and delete the spaces as needed. Clearly identify errors that cannot be auto-corrected. Provide an in-page link for easy access from the error to the field. Work with content writers to agree on content for instructions and messages.
+Be as forgiving of format as possible when processing user input. For example, accept phone numbers that include spaces and delete the spaces as needed. Clearly identify errors that cannot be auto-corrected. Work with content writers to agree on content for instructions and messages.
```

Comment:
I appreciate that "Provide an in-page link for easy access from the error to the field" isn't a requirement, but do you not think that it is a simple, helpful way to support the tip. My concern is that removal is watering down the tip content a little bit too much.